MEP_Engine/Compute/Mechanical/AirSide/ProcessLatentHeat.cs

```diff
@@ -0,0 +1,83 @@
+/*
+ * This file is part of the Buildings and Habitats object Model (BHoM)
+ * Copyright (c) 2015 - 2021, the respective contributors. All rights reserved.
```

Comment:
This should be set to 2022, but the auto update will run on the 3rd of January which means it'll then need updating to 2023 - so your choice which you go for depending on when you get back to look at this PR.